src/payment/data/sagas.js

```diff
@@ -154,7 +154,9 @@ export function* handleFetchCaptureKey() {
     yield put(microformStatus(STATUS_LOADING)); // we are refreshing the capture key
     const result = yield call(PaymentApiService.getCaptureKey);
     yield put(captureKeyDataReceived(result)); // update redux store with capture key data
-    yield put(captureKeyStartTimeout());
+    if (isWaffleFlagEnabled('payment.cybersource.flex_microform_enabled', false)) {
```

Comment:
I think this means that if you get the flag value from the server, rather than the url, then the timer won't start. In the other places that `isWaffleFlagEnabled` is used, I have it fall back to the value from the redux store, I think. Maybe we should just have one function for `isFlexMicroformEnabled` that checks redux and waffle?

Reply:
I had a hard time getting that from within the saga, but yes, if we can have one function for it, that would be ideal.

Reply:
Gonna go ahead and merge this so we can get the fix out to stop showing confusing messages in the old flow, we can look at unifying the enabled check later